library/init/meta/interactive.lean

```diff
@@ -975,13 +971,13 @@ private meta def delta_hyps : list name → list name → tactic unit
 | cs []      := skip
 | cs (h::hs) := get_local h >>= delta_hyp cs >> delta_hyps cs hs
 
-meta def delta : parse ident* → parse location → tactic unit
-| cs (loc.wildcard) := do ls ← tactic.local_context,
-                          n ← revert_lst ls,
-                          new_cs ← to_qualified_names cs,
-                          delta_target new_cs,
-                          intron n
-| cs l              := do new_cs ← to_qualified_names cs, l.apply (delta_hyp new_cs) (delta_target new_cs)
+meta def delta (cs : parse ident*) : parse location → tactic unit
+| loc.wildcard := do ls ← tactic.local_context,
+                     n ← revert_lst ls,
+                     new_cs ← to_qualified_names cs,
+                     delta_target new_cs,
+                     intron n
+| l            := do new_cs ← to_qualified_names cs, l.apply (delta_hyp new_cs) (delta_target new_cs)
 
```

Comment:
This looks like a random cosmetic change, and it is not related to the PR description.

Reply:
Sorry, I'll remove it.